src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private double radius;
+    private String color;
```

Comment:
we have this field from Figure, so we can remove color here

Reply:
solved, thank you